json/neutral_abilities.json

```diff
@@ -0,0 +1,122 @@
+{
+  "dark_troll_warlord_ensnare": {
+    "dname": "Ensnare",
+    "desc": "The Dark Troll Summoner tosses netting around an enemy's feet, briefly immobilizing the unit, though the unit can still attack and use abilities. Interrupts channeling abilities.", 
+    "cmb": "<div class=\"cooldownMana\"><div class=\"mana\"><img alt=\"Mana Cost\" title=\"Mana Cost\" class=\"manaImg\" src=\"http://cdn.dota2.com/apps/dota2/images/tooltips/mana.png\" width=\"16\" height=\"16\" border=\"0\" /> 150</div><div class=\"cooldown\"><img alt=\"Cooldown\" title=\"Cooldown\" class=\"cooldownImg\" src=\"http://cdn.dota2.com/apps/dota2/images/tooltips/cooldown.png\" width=\"16\" height=\"16\" border=\"0\" /> 20</div><br clear=\"left\" /></div>",
+    "dmg": "",
+    "affects": "ABILITY: <span class=\"attribVal\">Unit Target</span><br />AFFECTS: <span class=\"attribVal\">Enemy Units</span><br />PIERCES SPELL IMMUNITY: <span class=\"attribVal\"><font color=\"#70EA72\">Yes</font></span><br />",
+    "attrib": "RANGE: <span class=\"attribVal\">550</span><br />DURATION: <span class=\"attribVal\">1.75</span><br />",
+    "creepname": "Dark Troll Summoner",
+    "img": "/assets/images/dota2/neutral_abilities/dark_troll_warlord_ensnare.png"
```

Comment:
I think it'll try to use the img property.  If you don't commit the images nothing catastrophic will happen, but it'll show a broken image instead of the default "auto-attack" image.